### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards-Text Block-Color]: Colors on 'TextBlock' card elements page are not adapting to high contrast themes.

### ISSUE THREAD

**Issue by HasnuddinMohammed** [microsoft/AdaptiveCards#9156]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:** 
OS: Win11 
OS Version: 23H2 (OS Build 22631.5189) 
URL: https://adaptivecards.io/ 
Edge dev Browser version: 137.0.3255.0 (Official build) dev (64-bit) 
Screen reader: Narrator. 

**Pre requisites:** 
Turn on the High contrast mode form System settings > Accessibility > contrast theme   

**Repro Steps:** 
1.Open URL in the Edge dev browser & sign in with valid credentials 
2.Navigate and activate the Schema explorer. 
3.Activate the Text block in the left navigation under the Card Elements 
4.Scroll the page till color heading section and observe the issue. 

**Actual Result:** 
The card on the "Color" section has no colors when high contrast mode is used. 

**Expected Result:** 
The card on the "Color" section should adopt the colors when high contrast mode is used.

**Note:** Issue is repro throughout the page.

**User Experience:** 
Low vision users depends on the high contrast modes unable to identify the color if The card on the "Color" section has no colors when high contrast mode is used. 

 **MAS Reference:** 
[MAS 4.3.1 – No Disruption of Accessibility Features](https://aka.ms/MAS4.3.1)

### Screenshots

![Image](https://github.com/user-attachments/assets/ef11d988-bdec-4d8b-83ff-df17d91a7bc4)

![Image](https://github.com/user-attachments/assets/62b22451-9cf8-4688-bea4-7fe162878716)

![Image](https://github.com/user-attachments/assets/2271e84b-9712-41f9-a42f-77b2413047ba)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
As per the comment above closing this issue.
